### PR TITLE
Add SUS to Base

### PR DESCRIPTION
**Pull request by PlaguePatientZero**
Sustainable growth is a token focussed on an incetivized token locking mechanism to stimulate slow and steady ecosystem growth. Token is deployed on Base chain.

### CODE REVIEW

**Comment by PlaguePatientZero** [comment]
@roberto-bayardo

File: data/SUS/data.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Sustainable Growth",
+  "symbol": "SUS",
+  "decimals": 18,
+  "description": "Token focused at growing with a sustainable level due to incetivized temporarely locking mechanism.",
+  "tokens": {
```
Comment (roberto-bayardo) [inline data/SUS/data.json:6]: Please add the "nobridge": true tag since this is a native L2 token.

**Review by roberto-bayardo — CHANGES_REQUESTED** [review]

**Comment by hamdiallam** [comment]
cc @roberto-bayardo for another pass

**Comment by ericm-cb** [comment]
Hey @hamdiallam, can you guys explain 2 things:
1. What is the purpose of the LockedFunds in the transfer function? Will this at all impact the ability for a recipient to transfer/receive the full amount intended?

2. Why is the amount being set to 0 after the expired lock in the _beforeTokenTransfer function? Will this also impact the ability for a recipient to transfer/receive the full amount intended?

Thank you!

**Comment by PlaguePatientZero** [comment]
> Hey @hamdiallam, can you guys answer the following question:
> 
> 1. What is the purpose of the LockedFunds in the transfer function? Will this at all impact the ability for a recipient to transfer/receive the full amount intended?
> 
> Thank you!

Hi good question I made the code, you can look up the corresponding contracts of the token. The locked time is depending on the amount bought and equal to blockstowait = 150* sqrt(amount). In addition each wallet mints 20% tokens for free. Swapping works fine both ways it is just to inventive people locking longer for more value.

**Review by roberto-bayardo — APPROVED** [review]
approved for base

**Review by annieke — APPROVED** [review]

**Pull request merged**